offload/CMakeLists.txt

```diff
@@ -214,6 +226,31 @@ set(LIBOMPTARGET_GPU_LIBC_SUPPORT ${LLVM_LIBC_GPU_BUILD} CACHE BOOL
     "Libomptarget support for the GPU libc")
 pythonize_bool(LIBOMPTARGET_GPU_LIBC_SUPPORT)
 
+if(OPENMP_STANDALONE_BUILD)
+  if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    find_package(LLVM HINTS ${CMAKE_INSTALL_PREFIX})
+  endif()
+  find_path (
+    LIBOMP_OMP_TOOLS_INCLUDE_DIR
+    NAMES
+      omp-tools.h
+    HINTS
+    ${LLVM_INCLUDE_DIRS}
+    ${LLVM_INCLUDE_DIRS}/../lib/clang/${LLVM_VERSION_MAJOR}/include
```

Comment:
`LLVM_VERSION_MAJOR` won't be set if this doesn't go through the LLVM CMake. You'll need to call `--print-resource-dir` if the compiler is clang.

Reply:
It will be set from the LLVMConfig.cmake from the result of find_package(LLVM). I did query `clang --print-resource-dir` in a version before this but thought this was a cleaner way.